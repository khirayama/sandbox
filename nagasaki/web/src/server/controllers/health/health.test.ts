import { get } from '.';

test('should return the get of health api response', () => {
  const req: any = {};
  const res: any = {
    json: jest.fn()
  };

  get(req, res);

  expect(res.json).toHaveBeenCalled();
  expect(res.json.mock.calls[0][0]).toMatchSnapshot();
});
