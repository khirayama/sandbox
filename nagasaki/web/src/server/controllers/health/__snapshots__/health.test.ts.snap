// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the get of health api response 1`] = `
Object {
  "heath": "ok",
}
`;
